docs/src/main/asciidoc/mongodb-panache.adoc

```diff
@@ -765,6 +765,9 @@ MongoDB offers ACID transactions since version 4.0.
 
 To use them with MongoDB with Panache you need to annotate the method that starts the transaction with the `@Transactional` annotation.
 
+In MongoDB, a transaction is only possible on a replicaset,
+hopefully ou xref:mongodb.adoc#dev-services[Dev Services for MongoDB] setups a single node replicaset so it is compatible with transactions.
```

Comment:
`hopefully ou` is weird? Not sure what you wanted to say?

Reply:
Missing a 'r' for our